CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # `chainweb-node` Changelog
 
+## 2.18 (2023-03-01)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2023-02-23T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will expire on **2023-06-01T00:00:00Z**.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+[Changes](https://github.com/kadena-io/chainweb-node/compare/2.17.2...2.18):
+
+*   New /local endpoint preflight simulation API. (#1585, #1600)
+*   Support for Pact 4.6: (#1602)
+    *    New ZK native function support.
+    *    Better gas estimation.
+*   Warning deprecation system support in /local:
+    *    Allows the node to provide warnings for upcoming feature deprecations in Pact.
+    *    The same warnings are generated by the repl in Pact 4.6.
+*   Internal changes to support future chain database schema changes.
+*   Remove libtbb as a dependency.
```

Comment:
Do we have to update the README for this change?

Reply:
Already done